test/unit/mainController.test.ts

```diff
@@ -5,7 +5,7 @@
 
 import * as sinon from "sinon";
 import sinonChai from "sinon-chai";
-import { expect } from "chai";
+import { expect, assert } from "chai";
```

Comment:
We're trying to swap over to `expect` for everything because it reads more fluently.